### DISCUSSION TITLE
I need help because I can't resolve this error even putting sidetiltle

### DISCUSSION THREAD

**Discussion by pradofelipe** [imaNNeo/fl_chart#1609] [Q&A]
import 'package:flutter/material.dart';
import 'package:fl_chart/fl_chart.dart';

class WeightChart extends StatefulWidget {
  @override
  _WeightChartState createState() => _WeightChartState();
}

class _WeightChartState extends State<WeightChart> {
  List<Map<String, dynamic>>? chartData = [
    {"measured_at": "2022-01-01", "weight": 70.0},
    {"measured_at": "2022-02-01", "weight": 72.0},
    {"measured_at": "2022-03-01", "weight": 71.5},
    {"measured_at": "2022-04-01", "weight": 73.0},
    {"measured_at": "2022-05-01", "weight": 72.5},
    {"measured_at": "2022-06-01", "weight": 72.8},
    {"measured_at": "2022-07-01", "weight": 74.0},
  ]; // Lista para armazenar os dados do gráfico

  @override
  Widget build(BuildContext context) {
    return LineChart(
      LineChartData(
        minX: 0,
        maxX: (chartData!.length - 1).toDouble(), // Define o intervalo no eixo X

        // Configuração do gráfico...
        titlesData: FlTitlesData(
          // Define os rótulos para o eixo X
          bottomTitles: AxisTitles(
            showTitles: true,
            interval: 1,
            getTitles: (value) {
              if (value.toInt() < chartData!.length) {
                return chartData![value.toInt()]["measured_at"]
                    .toString()
                    .substring(5, 7); // Apenas o mês
              }
              return '';
            },
          ),
          // Define os rótulos para o eixo Y
          leftTitles: AxisTitles(
            showTitles: true,
            interval: 1,
            getTitles: (value) {
              return value.toInt().toString();
            },
          ),
        ),
        gridData: FlGridData(show: true),
        borderData: FlBorderData(show: true),
        lineBarsData: [
          LineChartBarData(
            spots: List.generate(
              chartData!.length,
              (index) {
                // Cria um ponto no gráfico para cada medida de peso
                final weight = double.parse(chartData![index]["weight"].toString());
                return FlSpot(index.toDouble(), weight);
              },
            ),
            isCurved: true,
            colors: [Colors.blue],
            barWidth: 4,
            isStrokeCapRound: true,
            belowBarData: BarAreaData(show: false),
          ),
        ],
      ),
    );
  }
}

**Comment by imaNNeo** [comment]
Please provide more information. And a reproducible code (a main.dart file content)
